.github/workflows/ci_reusable_wf.yml

```diff
@@ -0,0 +1,404 @@
+# You can find more information about the design decisions here:
```

Comment:
We can move this to the bors CI as the referring PR was mostly about how we test the Android and iOS libraries